### PR TITLE
Optimize logging in rdzf manager.

### PR DESCRIPTION
**Pull request by BalaBalaYi**
### What changes were proposed in this pull request?

Improve logging.

### Why are the changes needed?

To improve the logging for better tracing.

### Does this PR introduce any user-facing change?

No.

### How was this patch tested?

UT.

### CODE REVIEW

File: dlrover/python/master/elastic_training/rdzv_manager.py [hunk 1]
```diff
@@ -174,9 +174,40 @@ def _check_rdzv_completed(self):
...
+            logger.info(
+                f"Waiting nodes(required:{self._rdzv_params.min_nodes}"
+                f"/{self._rdzv_params.max_nodes}) in rendezvous(size:"
+                f"{len(waiting_nodes)}) are {waiting_nodes}, "
+                f"lacking ranks(size:{len(lacking_ranks)}) "
+                f"are {lacking_ranks}"
+            )
         return rdzv_completed
 
+    def _get_lacking_ranks(self):
```
Comment (samplise) [inline dlrover/python/master/elastic_training/rdzv_manager.py:187]: `def _get_lacking_ranks(self) -> List(int):`

File: dlrover/python/master/elastic_training/rdzv_manager.py [hunk 2]
```diff
@@ -174,9 +174,40 @@ def _check_rdzv_completed(self):
...
+        """
+        Lacking ranks = min required nodes - waiting ranks.
```
Comment (samplise) [inline dlrover/python/master/elastic_training/rdzv_manager.py:189]: min required nodes -> min required ranks

File: dlrover/python/master/elastic_training/rdzv_manager.py [hunk 3]
```diff
@@ -174,9 +174,40 @@ def _check_rdzv_completed(self):
...
+
+        Return:
+            ranks number in list
```
Comment (samplise) [inline dlrover/python/master/elastic_training/rdzv_manager.py:192]: ranks in list

File: dlrover/python/master/elastic_training/rdzv_manager.py [hunk 4]
```diff
@@ -575,7 +608,9 @@ def check_fault_node(self):
                     fault_nodes = {}
                     for rank in self._fault_nodes:
                         fault_nodes[rank] = self._rdzv_nodes[rank].node_id
-                    logger.warning(f"Fault nodes are {fault_nodes}")
+                    logger.warning(
+                        "Fault nodes(rank:node_id) " f"are: {fault_nodes}"
```
Comment (workingloong) [inline dlrover/python/master/elastic_training/rdzv_manager.py:612]: Merge the two string to one.

**Review by samplise — APPROVED** [review]

**Pull request merged**